Snowplow iOSTests/TestUtils.m

```diff
@@ -90,6 +90,11 @@ - (void)testGetTimestamp {
 
 }
 
+- (void) testTimestampToISOString {
+    XCTAssertEqualObjects([SPUtilities timestampToISOString:1654496481347], @"2022-06-06T06:21:21.347Z");
```

Comment:
Is this time too precise? Not sure about that final `.347` bit. [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601).

Reply:
"Looks like timestamps are more complicated than I realised!" – I tell that to myself every time I work with timestamps 😀